app/plugins/router.js

```diff
@@ -1,51 +1,51 @@
 const config = require('../config')
 
 let routes = [].concat(
```

Comment:
couldn't this just be an array? Not sure why it's using concat.

Reply:
Although if we're changing them to not be arrays in the future, maybe its worth just fixing this when that happens?

Reply:
This PR is a bit of a halfway house sort of piece of work - it enables us to export some functions so we can unit test them, but realistically we need to move to ES modules before we do the proper refactoring I think, because so much will change when we do that.

Reply:
a right yes the concat will flatten the first nested array 👍